WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -832,6 +834,14 @@ - (BlogDetailsSection *)jetpackSectionViewModel
                                                      }]];
     }
 
+    if ([self.blog supports:BlogFeatureJetpackScan]) {
```

Comment:
Since we're merging that to `develop` we should create a Feature Flag for it, no?

Reply:
Oops 🤦‍♀️, done in: 540bc33727df1aee015fb80e276a57529289720b